.github/workflows/tests.yml

```diff
@@ -71,3 +71,7 @@ jobs:
       - name: yarn test:schemas
         if: matrix.node-version == '10.x'
         run: yarn test:schemas
+
+      - name: yarn test:compat
+        if: matrix.node-version == '10.x'
```

Comment:
Since you're here, can you take care of https://github.com/microsoft/botbuilder-js/projects/4#card-49104042

Reply:
Resolution would be to remove Node 10 from the matrix and then rebalance the jobs we've distributed across node versions.

Reply:
Do we want to add 15.x or 16.x at all, or just have 12.x and 14.x?

Reply:
You could try adding 16.x, there may be an issue with one or more of our dependencies though.

Reply:
16.x currently blocked by Orchestrator Core

![image](https://user-images.githubusercontent.com/40401643/118868926-a274c200-b899-11eb-9c72-a6edd4869d0a.png)

Reply:
Thought so - thanks for checking!